### DISCUSSION TITLE
MacOS latest working Version 3.1.1.

### DISCUSSION THREAD

**Discussion by bjoernCCG** [sn4k3/UVtools#447] [General]
Hello everyone,

First of all, big thanks for the effort which went into this project. Highly appreciated!

Does anyone else have the problem that the releases after 3.1.1 don't start. MacOs just prompts that its unable to open it. 
To be clear: I don't refer to the "verified developer" prompt. This also comes up at 3.1.1. but can be approved.
3.2 and 3.2.1 don't open at all neither x64 nor arm64 Version. Tested on installation via Home-brew and Download. In Finder and Terminal.

I can work with 3.1.1 but I'm curious whether I'm doing something wrong.

**Comment by sn4k3** [comment]
Try to remove **UVtools.entitlements** from the **UVtools.app** folder, does it work?

"verified developer" never show as an error, it will always say no permitions or other unclear error

**Reply by bjoernCCG** [reply]
Regarding "verified developer" - yes, I tried to open it wit right click and also open the security setting of macOs where these usually show up. No problems with 3.1.1

**Reply by bjoernCCG** [reply]
Found it. The UVTolls file didn't have the execution right in the 3.2.1. version. chmodded it, now it seems to work

**Reply by sn4k3** [reply]
Yes you need to chmod, for some reason it is losing the permissions. at least +x

**Comment by dinobaurier** [comment]
Unfortunately no difference
<img width="297" alt="Bildschirmfoto 2022-04-04 um 12 13 28" src="https://user-images.githubusercontent.com/97173171/161523395-aa59cf1b-c95b-4ab2-8628-725c3503c62b.png">

**Reply by dinobaurier** [reply]
Sorry, private account - but user bjoernCCG
